packages/snfoundry/contracts/src/test/TestContract.cairo

```diff
@@ -1,14 +1,14 @@
 use contracts::YourCollectible::{IYourCollectibleDispatcher, IYourCollectibleDispatcherTrait};
-use contracts::components::ERC721Enumerable::{
-    IERC721EnumerableDispatcher, IERC721EnumerableDispatcherTrait
+use openzeppelin_token::erc721::extensions::ERC721EnumerableComponent::{
```

Comment:
@gianalarcon Do you know the correct path to import the dispatcher and trait for this OZ component? The one I set here is not working

Reply:
@Nadai2010 Do you have any idea?